bin/inx-chronicle/src/metrics.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{convert::Infallible, sync::Arc};
+
+use async_trait::async_trait;
+use bee_metrics::{encoding::SendSyncEncodeMetric, serve_metrics, Registry};
+use chronicle::runtime::actor::{context::ActorContext, event::HandleEvent, Actor};
+use futures::future::FutureExt;
+use tokio::task::JoinHandle;
+
+#[derive(Default)]
+pub struct MetricsWorker {}
+
+#[async_trait]
+impl Actor for MetricsWorker {
+    type State = (Arc<Registry>, JoinHandle<()>);
+
+    type Error = Infallible;
+
+    async fn init(&mut self, _cx: &mut ActorContext<Self>) -> Result<Self::State, Self::Error> {
+        let registry = Arc::new(Registry::default());
+
+        // FIXME: pass address via config.
+        let addr = "0.0.0.0:6969".parse().unwrap();
+        let fut = tokio::spawn(serve_metrics(addr, registry.clone()).map(|res| res.unwrap()));
```

Comment:
We should await this future, so I would spawn a task to do that and send a shutdown signal. See the API actor for an example.